server/rest/submission.py

```diff
@@ -253,12 +253,7 @@ def postSubmission(self, phase, folder, params):
                     'headers': {'Girder-Token': scoreToken['_id']}
                 }
             },
-            'jobInfo': {
-                'method': 'PUT',
-                'url': '/'.join((apiUrl, 'job', str(job['_id']))),
-                'headers': {'Girder-Token': jobToken['_id']},
-                'logPrint': True
-            },
+            'jobInfo': utils.jobInfoSpec(job, token=jobToken),
```

Comment:
The nice thing about using this helper function is that it will internally create the jobToken for us, so we no longer have to create it in this method.